.circleci/config.yml

```diff
@@ -0,0 +1,67 @@
+# Golang CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-go/ for more details
+version: 2
+jobs:
+  build:
```

Comment:
are we using directly the host machine to build everything instead of some of the prepared circleci images?

Reply:
We are using a VM instance here since we need Docker. While CircleCI offers a remote Docker solution it doesn't allow to share files or networking this unable to run bblfsh. 
The VM instance can only run a few builds of Ubuntu Trusty with an old version of Go.

Reply:
More info: https://github.com/src-d/gitbase/pull/652#issuecomment-239064951

Reply:
More info https://github.com/src-d/infrastructure/issues/209#issuecomment-440328053